tee-worker/omni-executor/rpc-server/src/request.rs

```diff
@@ -22,6 +22,12 @@ use executor_primitives::ShardIdentifier;
 use parity_scale_codec::{Decode, Encode};
 use std::fmt::Debug;
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug)]
+pub struct PlainRequest {
+	pub shard: ShardIdentifier,
```

Comment:
Why do we need this ?

Reply:
oh, I forgot to check the mrenclave. thanks 👍🏼

Reply:
I mean, why do we need `shard` at all :) 
Is there any sharding, because i'm a little bit confused - maybe this should be named differently?

Reply:
ah yes. Well, initially it was discussed that we wanted to be as close as possible to the identity worker and reuse as much as possible the old code/logic so I've been trying not to change to many things. I guess we can just call it `mrenclave` and if we ever need to support sharding we can just refactor it

Reply:
It's scale encoded so name doesn't matter for clients

Reply:
true